eZ/Publish/Core/SignalSlot/SectionService.php

```diff
@@ -197,6 +199,29 @@ public function assignSection(ContentInfo $contentInfo, Section $section)
         return $returnValue;
     }
 
+    /**
+     * Assigns the subtree to the given section
+     * this method overrides the current assigned section.
```

Comment:
Nitpick: You probably meant this two lines as two separate sentences. Please then place dot and at the end of first one and start the next one with capital letter. Otherwise on PhpDoc preview it looks like one sentence :)